src/IssueViz/IssueVisualizationControl/IssueVisualizationControl.xaml

```diff
@@ -103,6 +103,24 @@
             </Style.Triggers>
         </Style>
 
+        <!-- Navigability warning footer -->
+
+        <Style x:Key="NavigabilityWarningTextBlockStyle" TargetType="TextBlock" BasedOn="{StaticResource BaseTextBlockStyle}">
+            <Setter Property="Foreground" Value="{DynamicResource {x:Static vsShell:VsBrushes.InfoTextKey}}"/>
+            <Setter Property="FontStyle" Value="Italic"/>
+        </Style>
+
+        <Style x:Key="NavigabilityWarningBorderStyle" TargetType="Border">
```

Comment:
The new control is a border that contains the icon and the text.
The entire border has a background color as the VS goldbar.